sklearn_pandas/categorical_imputer.py

```diff
@@ -0,0 +1,73 @@
+"""
+
+Impute missing values from a categorical/string np.ndarray or pd.Series with the most frequent value on the training data.
+
+"""
+
+import pandas as pd
+import numpy as np
+
+from sklearn.base import TransformerMixin
+
+
+class CategoricalImputer(TransformerMixin):
+
+    """
+
+    Attributes
+    ----------
+
+    fill : str
+        Most frequent value of the training data.
+
+    """
+
+    def __init__(self):
+
+        self.fill = None
+
+    def fit(self, X):
+
+        """
+
+        Get the most frequent value.
+
+        Parameters
+        ----------
+            X : np.ndarray or pd.Series
+                Training data.
+
+        Returns
+        -------
+        CategoricalImputer
+            Itself.
+
+        """
+
+        self.fill = pd.Series(X).mode().values[0]
```

Comment:
This is implicitly assuming that there will be only one mode value. Can you raise an explicit exception if this is not true? Something like:

```
modes = pd.Series(X).mode()
if modes.shape[0] == 0:
    raise ValueError('No value is repeteated more than twice in the column')
elif modes.shape[0] > 1:
    raise ValueError('Column has multiple modes {}, can't select one to fill'.format(modes.tolist())
else:
    self.fill = modes[0]
```